src/App.js

```diff
@@ -1,24 +1,42 @@
-import logo from './logo.svg';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link,
+} from 'react-router-dom';
+import { HiUser } from 'react-icons/hi';
 import './App.css';
+import Books from './components/Books';
+import Categories from './components/Categories';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <header>
+          <div className="leftNav">
+            <h1>BookStore THC</h1>
+            <ul>
+              <li>
+                <Link to="/">Books</Link>
+              </li>
+              <li>
+                <Link to="/categories">Categories</Link>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <HiUser color="blue" fontSize="1.5rem" />
+          </div>
+        </header>
```

Comment:
- [ ] **[OPTIONAL]** I think section is intended as your Navbar. It would probably be nice to extract this into a new component and render that component here.